### DISCUSSION TITLE
五月天、阿信魅力無法擋！靠一招就能控制全場

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#959] [General]
當然，大家都知道五月天的魅力無法擋，但在這次的諾亞方舟復刻演唱會中，他們使用到了物聯網的技術，在五月天特製的互動螢光棒裡裝有小小的無線接收器，當控制台發出無線電波訊號，螢光棒就會依指令變色，讓全場氣氛更加熱烈。物聯網現在也是一個很熱門的技術，也能應用在演唱會中，帶動演唱會氣氛，真的好酷!
(好想去看五月天演唱會，聽說今年要在高雄加開4場耶!)

https://www.businesstoday.com.tw/article/category/80394/post/201812260006/

**Comment by AnAnistired** [comment]
五月天演唱會的螢光棒真的超屌，重點是他很久以前就已經出現了，可以控制全場螢光棒的演唱會真的很少見，不知道他的技術提供是哪一家廠商？

**Reply by moonnight3503** [reply]
真的超酷，我也好想現場體驗一下在五月天的演唱會用那特製的螢光棒!

**Comment by Floriaand** [comment]
最近五月天在桃園辦了好幾場演唱會，1/7、8還有場次，樓主也可以考慮在桃園聽。

**Reply by moonnight3503** [reply]
可是他的票真的好難好難搶QAQ，開賣才沒多久就被掃光了QQ

**Comment by ru0372572** [comment]
這好酷！感覺很漂亮～～～

**Reply by moonnight3503** [reply]
超漂亮~<3

**Comment by Evan0979** [comment]
科技可以用在各種地方,一點小巧思就能成為引人注目的創新!

**Comment by swerfg03joe** [comment]
物聯網在演唱會的小應用，能做到這麼廣泛的控制，也說明5G覆蓋率相當高

**Comment by peggy0329** [comment]
我覺得這樣的演唱會體驗設計的很用心
聲光效果對演唱會來書很重要
結合技術讓聽眾體驗更好是一個很棒的應用

**Comment by dinosaur1009** [comment]
把科技與演唱會連結，打造更好的體驗！很棒的應用

**Comment by dabbie109303501** [comment]
看完文章好想買導航鞋喔 這對google map使用障礙者是一大福音ㄝ 
btw我也好想去五月天演唱會 但沒$只能聽網路版過過癮QAQ

**Reply by moonnight3503** [reply]
哈哈哈我也是聽網路版的ㄟ，每年跨年都是跟五月天一起跨

**Comment by HuangHsinTzu** [comment]
我今年也有去聽林俊傑的演唱會
現場有免費發放螢光棒~
但因為是免費的就沒有五月天演唱會的高級哈哈
這感覺超酷的欸
只是聽說一支不便宜欸

我個人是蠻看好物聯網的發展
尤其是智慧家電的部分
隨著家庭結構越來越小及雙薪家庭越來越多
智慧家電真的是很棒的技術!

**Comment by BlahFish1006** [comment]
我聽朋友說五月天自己現在也有個APP，是可以在自己家裡參加線上演唱會的時候，
控制螢光棒也能跟著現場一起變色！
讓線上演唱會更有臨場感呢！

**Comment by Sylvie0601** [comment]
韓國從幾年前開始就已經都會這樣了!韓國IDOL都有各自的應援棒，每個應援棒都會有適配的APP可以下載，然後當要入場看演唱會的話，透過掃描演唱會的QRcode就可以由主控台控制每一個粉絲的應援棒，然後配合舞台燈光憶起做出變化 ! 而且不管現場還是在線上演唱會都有這種QRcode可以掃，或是也可以透過自己的APP手動變化燈光效果，真的超級酷 !

**Comment by Tuoresa** [comment]
之前都是看韓國的演唱會有全場一起變化的燈海
五月天竟然也有了！太酷了好想去現場

現在演唱會真的因為科技越來越卓越！
那天五月天在桃園演唱會的嘉賓甚至用3D全息投影的方式出現，
阿信沒說之前，現場有些觀眾以為是真人來到現場哈哈哈哈

**Reply by moonnight3503** [reply]
喔喔!對對!是鄧紫棋的那一場，我一開始也沒有注意到
後來才發現紫棋跟阿信的比例有點怪
真的超級酷的，但感覺他們辦一場演唱會就很燒錢
而且還要連續唱那麼多天，真的好辛苦

**Comment by Valentinaa0214** [comment]
螢光棒隨著歌詞變色超酷的！
感覺把觀眾席的範圍也融入到演唱會的表演中了，現場看一定超震撼！

**Comment by bonny123865** [comment]
感覺他們辦演唱會好用心!(但是也好燒錢)
沒想到科技還能運用在歌迷跟偶像的互動上，太優秀了吧!

**Comment by tzurou** [comment]
五迷路過~已經連續兩年都去五月天演唱會，也有買螢光棒，五月天演唱會厲害的地方真的是燈光，現場燈暗下來幾萬隻螢光棒真的很漂亮，今年又多增加座位綁定會根據坐的位置有不一樣顏色的光，只能說主辦方真的很用心在辦每一場演唱會讓我每年都想去！

**Comment by sunset513** [comment]
我自己也是個五迷，他的歌真的是從小聽到大，陪著我一起成長！

我也有去過演唱會，我覺得現場的版本真的很不一樣，整個人的情緒會被渲染，當我聽到熟悉的歌曲時更是感動。雖然阿信會有些失誤的哈哈哈，但我覺得這樣更有感覺。
